src/Confluent.Kafka/Timestamp.cs

```diff
@@ -80,6 +88,15 @@ public static long DateTimeToUnixTimestampMs(DateTime dateTime)
             }
         }
 
+        /// <summary>
+        ///     Convert a unix timestamp to a DateTime value.
+        /// </summary>
+        /// <param name="timestamp">
+        ///     The unix timestamp to convert.
+        /// </param>
+        /// <returns>
+        ///     The DateTime value associated with <paramref name="timestamp"/>
+        /// </returns>
         public static DateTime UnixTimestampMsToDateTime(long timestamp)
             => new DateTime(1970, 1, 1, 0, 0, 0, 0, System.DateTimeKind.Unspecified)
```

Comment:
@edenhill I was considering timestamps reliant on the move to 0.9.4 (remembering the bug in 0.9.2 i mentioned), so had been putting off testing until now. Will do some unit an integration tests on master (master).

@treziac I'm not convinced by DateTimeOffset - I think we should keep everything as DateTime - DateTimeOffset seems a more complex format than strictly necessary.